scripts/build.ckusdc_ledger.args.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PRINCIPAL="$(dfx identity get-principal)"
+ARG_FILE="$(jq -re .canisters.ckusdc_ledger.init_arg_file dfx.json)"
+
+mkdir -p "$(dirname "$ARG_FILE")"
+
+cat <<-EOF >"$ARG_FILE"
+(variant {
+	Init = record {
+		 token_symbol = "ckSepoliaUSDC";
+		 token_name = "Chain key Sepolia USDC";
+		 decimals = opt 6;
+		 max_memo_length = opt 80;
+		 minting_account = record { owner = principal "$CANISTER_ID_CKETH_MINTER" };
+		 transfer_fee = 9_500;
+		 metadata = vec {};
+		 initial_balances = vec {
+		   record { record { owner = principal "$PRINCIPAL"; }; 100_000_000_000_000_000_000; };
+		   record { record { owner = principal "x4w27-so7wg-cudsa-yy7fh-wcpy5-njul4-q54tv-euzzi-tdnzz-ill46-zqe"; }; 500_000_000_000_000_000; };
```

Comment:
Note: This argument has been moved here from `scripts/deploy.ckerc20.sh`.  I do not know what the `4xw...` principal is.